### ISSUE TITLE
Is this still in developement?

### ISSUE THREAD

**Issue by nomatica** [bryanph/GeistMap#184]
Stumbled upon this and was curious if it was still in development

**Comment by iammeat** [comment]
[This article](https://hackernoon.com/building-a-open-source-personal-knowledge-base-45c25f5a4324) was posted two weeks ago, which is strange.  No movement on the project for some time.  @bryanph I also noticed the twitter link on that article links to a non-existent user.

**Comment by bryanph** [comment]
TLDR; No, but I'm still working on the problem in another incarnation of this project: https://topictrails.com/

This project started out as an experiment to find new ways to organize research and study material because I was very frustrated with how difficult it is to keep notes and extract quotations and references from source material. Over time my focus has kind of shifted from more of a learning system, where you wish to organize facts that are out there, to a research system where you wish to collect arguments and counter-arguments for the research you conduct. I found that the two are quite different in practice due to the objectiveness of the information you collect in the second.

Therefore I started a new iteration of this project which goes in a slightly different direction. Whilst in Geistmap my focus was on connecting information and facts in a visual graph-like structure, in Topictrails the focus is on extracting information from other sources (such as PDF and web pages) and being able to reference any part of that source, either embedding (fully or partially) it directly in a new document or referencing a certain section. A lot of these things are possible (more or less) today when you combine different tooling but my goal here is to absolutely minimize the amount of friction it takes to do these actions so that it is easy to create habits around them. 

When using Geistmap I found myself wanting to reference content within other nodes which was not easy because of the way Geistmap represents its data (as full nodes that you reference). Therefore I decided to go for more of a traditional document-like approach. Just like in Geistmap, you can reference other nodes (documents) in Topictrails but there is more of a focus on creating deep connections to the exact section you wish to reference. As the name "Topictrails" suggests, this means you often end up with a "trail" of references, each reference bringing you to another source document. The goal here is to make this navigation as smooth as possible, solving the difficult problem of referencing external sources and maintaining references in your own writing and research.

@iammeat I have no idea why that article has been reposted but I wrote that article more than 2 years ago. Thanks for bringing it to my attention, I will try and make sure it goes down.

As for Twitter, you can follow me here: https://twitter.com/bryanph_

**Comment by iammeat** [comment]
Thanks very much for replying in such a detailed fashion, @bryanph.  

This lets me understand more about what you were looking for in terms of scope and referencing.  I do think that Geistmap still has an extremely strong use-case, as I've been searching for a personal wiki since the early days of ZuluPad that would do what you've almost perfected here.  Evernote and other note-taking tools are too linear and don't offer interlinking, and they're getting worse with new features that people just don't want.  I mean, Microsoft OneNote is 1.08GB... for a note taking app!

Topictrails seems more of a deep referencing tool, and I can see that many aspects of that would also add value to Geistmap (PDF extraction, etc.).

I would be interested in contributing to the project if you would be open to that, but I don't want to drag it away from your own mindset by doing so.  I have forked the project, and I can always submit PRs if you see anything that would fit in with what you see as Geistmap's modus operandi.

**Comment by StoltHD** [comment]
> 
> 
> Thanks very much for replying in such a detailed fashion, @bryanph.
> 
> This lets me understand more about what you were looking for in terms of scope and referencing. I do think that Geistmap still has an extremely strong use-case, as I've been searching for a personal wiki since the early days of ZuluPad that would do what you've almost perfected here. Evernote and other note-taking tools are too linear and don't offer interlinking, and they're getting worse with new features that people just don't want. I mean, Microsoft OneNote is 1.08GB... for a note taking app!
> 
> Topictrails seems more of a deep referencing tool, and I can see that many aspects of that would also add value to Geistmap (PDF extraction, etc.).
> 
> I would be interested in contributing to the project if you would be open to that, but I don't want to drag it away from your own mindset by doing so. I have forked the project, and I can always submit PRs if you see anything that would fit in with what you see as Geistmap's modus operandi.

Just to your information @iammeat Onenote has interlink between both Notes and sections in Notes... and I think one of the reasons its so big is because of the OCR...

But I agree that it lack a lot of other things, lig a graphical view of those linke as a network/knowledge graph...

I also agree that this project looked really promising, I'm also hunting for a knowledge graph software solution that I can both use localy and natively, and deplay as a personal web service, and be able to sync (or use the same database).
Best of all would be iof the soaftware also used a multi store database or at least a graph database...

One of the most important things is that it must be possible to install and use the system localy...
I actually are thinking about installing a heavy duty system like Blue Brain Nexus to get some of what I want...

**Comment by bryanph** [comment]
@iammeat Yes the two more or less complement one another indeed. It's just that the subject area is so big that in order to reach any goal I have to narrow down.

You are of course very welcome to fork the project and expand on it as you see fit. If you create PRs I can also take a look at them and if you want to work on it for longer term I can also make you a collaborator on the repository if you wish.
